html/browser_action/views/password_prompt.html

```diff
@@ -5,7 +5,7 @@
             <div>{{'extension_locked'| translate}}:</div>
             <div class="group">
                 <input autofocus type="password" ng-enter="apply_settings()" placeholder="{{'password' | translate}}"
-                       ng-model="master_password">
+                       id="master_password" ng-model="master_password">
```

Comment:
why do you give it an id? is it required by anything?

Reply:
No ID required. Will be removed again.